### PR TITLE
230 set up repo

### PR DESCRIPTION
**Pull request by katjam**
https://trello.com/c/atXhHCs8/230-vvv-e-set-up-repo

- init repo with readme, license and gitignore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
-# sea-map
+# SEA-MAP Tech vs. Abuse project
+
+### Site URLs
+- [Alpha](http://neontribe.github.io/sea-map)
+
+### Development
```
Comment (Nikomus) [inline README.md]: Should this be ## and the headings beneath ### ?

Comment (katjam) [reply README.md]: lol. Yes. I need your plugin!

Comment (Nikomus) [reply README.md]: 😅 https://github.com/JamshedVesuna/vim-markdown-preview maybe

**Review by Nikomus — APPROVED** [review]
All looks reasonable to me ✔️

**Pull request merged**